practice/pyramid/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Pyramid</title>
+    <link rel="stylesheet" href="site.css">
+  </head>
+  <body>
+    <div class="row">
+      <div class="inner">
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in.
+        </p>
```

Comment:
Sorry if the problem statement wasn't clear. I wanted you to exactly re-create the sample page using just flex box. So there should only be `div`s and identical borders.